### ISSUE TITLE
Vanilla Lucata LAGraph Tests

### ISSUE THREAD

**Issue by skuntz** [emusolutions/LAGraph#1]
I think this is the current set of "Vanilla" LAGraph tests and all should pass except connected components and types.
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/skuntz/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/skuntz/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

test_acutest
--
test_Betweenness
test_BreadthFirstSearch
test_Cached_AT
test_Cached_Degree
test_Cached_NDiag
test_Cached_SymmetricStructure
test_CheckGraph
test_ConnectedComponents
test_DeleteCached
test_DisplayGraph
test_export
test_fopen
test_Init
test_Init_errors
test_IsEqual
test_KindName
test_Malloc
test_Matrix_Structure
test_minmax
test_MMRead
test_Multiply_size_t
test_New
test_NumThreads
test_PageRank
test_SampleDegree
test_SingleSourceShortestPath
test_Sort
test_SortByDegree
test_TriangleCount
test_Type
test_vector
test_Vector_Print
test_Vector_Structure
test_WallClockTime
test_Xinit

</body>

</html>

**Comment by jamesETsmith** [comment]
Just wanted to follow up on this for two reasons to have a record of what's passing for x86 so we know how LC builds do in comparison. 

TLDR `test_ConnectedComponents` is the only test that's failing for x86, which makes sense because it requires user-defined select ops which we don't implement.

### LucataGraphBLAS:
- hash: 0efd7d2a
- cmake configuration: 
```
cmake -S . -B build -DCMAKE_TOOLCHAIN_FILE=cmake/x86-toolchain.cmake -DCMAKE_BUILD_TYPE=Release
```

### LAGraph
- hash: 63297e58
- cmake configuration:
```
cmake -B build -DGRAPHBLAS_ROOT=/net/hyper120h-d/data/jsmith/apps/LucataGraphBLAS/build/install
```

## Results
```
❯ ctest --test-dir build --parallel 12
Internal ctest changing into directory: /net/hyper120h-d/data/jsmith/apps/LAGraph/build
Test project /net/hyper120h-d/data/jsmith/apps/LAGraph/build
      Start  1: ctest_Betweenness
      Start  2: ctest_BreadthFirstSearch
      Start  3: ctest_Cached_AT
      Start  4: ctest_Cached_Degree
      Start  5: ctest_Cached_NDiag
      Start  6: ctest_Cached_SymmetricStructure
      Start  7: ctest_CheckGraph
      Start  8: ctest_ConnectedComponents
      Start  9: ctest_DeleteCached
      Start 10: ctest_DisplayGraph
      Start 11: ctest_Init
      Start 12: ctest_Init_errors
 1/36 Test  #1: ctest_Betweenness .................   Passed    0.01 sec
      Start 13: ctest_IsEqual
 2/36 Test  #3: ctest_Cached_AT ...................   Passed    0.01 sec
      Start 14: ctest_KindName
 3/36 Test  #4: ctest_Cached_Degree ...............   Passed    0.01 sec
      Start 15: ctest_MMRead
 4/36 Test  #5: ctest_Cached_NDiag ................   Passed    0.01 sec
      Start 16: ctest_Malloc
 5/36 Test  #7: ctest_CheckGraph ..................   Passed    0.01 sec
      Start 17: ctest_Matrix_Structure
 6/36 Test  #8: ctest_ConnectedComponents .........***Failed    0.01 sec
      Start 18: ctest_Multiply_size_t
 7/36 Test #11: ctest_Init ........................   Passed    0.01 sec
      Start 19: ctest_New
 8/36 Test #10: ctest_DisplayGraph ................   Passed    0.01 sec
      Start 20: ctest_NumThreads
 9/36 Test #12: ctest_Init_errors .................   Passed    0.01 sec
      Start 21: ctest_PageRank
10/36 Test #13: ctest_IsEqual .....................   Passed    0.01 sec
      Start 22: ctest_SampleDegree
11/36 Test #14: ctest_KindName ....................   Passed    0.01 sec
      Start 23: ctest_SingleSourceShortestPath
12/36 Test #16: ctest_Malloc ......................   Passed    0.01 sec
      Start 24: ctest_Sort
13/36 Test #17: ctest_Matrix_Structure ............   Passed    0.01 sec
      Start 25: ctest_SortByDegree
14/36 Test #18: ctest_Multiply_size_t .............   Passed    0.01 sec
      Start 26: ctest_TriangleCount
15/36 Test #19: ctest_New .........................   Passed    0.01 sec
      Start 27: ctest_Type
16/36 Test #20: ctest_NumThreads ..................   Passed    0.01 sec
      Start 28: ctest_Vector_Print
17/36 Test #21: ctest_PageRank ....................   Passed    0.01 sec
      Start 29: ctest_Vector_Structure
18/36 Test #22: ctest_SampleDegree ................   Passed    0.01 sec
      Start 30: ctest_WallClockTime
19/36 Test  #2: ctest_BreadthFirstSearch ..........   Passed    0.04 sec
      Start 31: ctest_Xinit
20/36 Test #15: ctest_MMRead ......................   Passed    0.03 sec
      Start 32: ctest_acutest
21/36 Test #25: ctest_SortByDegree ................   Passed    0.02 sec
      Start 33: ctest_export
22/36 Test #27: ctest_Type ........................   Passed    0.02 sec
      Start 34: ctest_fopen
23/36 Test #28: ctest_Vector_Print ................   Passed    0.02 sec
      Start 35: ctest_minmax
24/36 Test #29: ctest_Vector_Structure ............   Passed    0.02 sec
      Start 36: ctest_vector
25/36 Test  #9: ctest_DeleteCached ................   Passed    0.04 sec
26/36 Test #31: ctest_Xinit .......................   Passed    0.01 sec
27/36 Test #32: ctest_acutest .....................   Passed    0.01 sec
28/36 Test #33: ctest_export ......................   Passed    0.00 sec
29/36 Test #34: ctest_fopen .......................   Passed    0.00 sec
30/36 Test #36: ctest_vector ......................   Passed    0.00 sec
31/36 Test #23: ctest_SingleSourceShortestPath ....   Passed    0.04 sec
32/36 Test #35: ctest_minmax ......................   Passed    0.02 sec
33/36 Test  #6: ctest_Cached_SymmetricStructure ...   Passed    0.07 sec
34/36 Test #24: ctest_Sort ........................   Passed    0.12 sec
35/36 Test #30: ctest_WallClockTime ...............   Passed    0.12 sec
36/36 Test #26: ctest_TriangleCount ...............   Passed  2131.62 sec

97% tests passed, 1 tests failed out of 36

Total Test time (real) = 2131.65 sec

The following tests FAILED:
          8 - ctest_ConnectedComponents (Failed)
Errors while running CTest
Output from these tests are in: /net/hyper120h-d/data/jsmith/apps/LAGraph/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
```

I don't remember `test_TriangleCount` taking so long before, it seems like something's off here.
